src/_pytest/cacheprovider.py

```diff
@@ -5,39 +5,41 @@
 ignores the external pytest-cache
 """
 from __future__ import absolute_import, division, print_function
-
 from collections import OrderedDict
 
 import py
 import six
+import attr
 
 import pytest
 import json
-import os
 from os.path import sep as _sep, altsep as _altsep
+import shutil
+
+from . import paths
```

Comment:
I think you forgot to include `paths.py` into the commit.

Reply:
fixed